mod/metrics/set/metric_type/relationship.rb

```diff
@@ -23,14 +23,21 @@ def researched?
   add_title_inverse_pointer metric_title, inverse
 end
 
-def new_inverse_title
-  inverse_field = subfield(:inverse_title)
-  inverse_field&.content.present? && inverse_field.content
-end
-
-def add_title_inverse_pointer title, inverse
-  add_subcard [inverse, :inverse], content: title, type_id: Card::PointerID
-  add_subcard [title, :inverse], content: inverse, type_id: Card::PointerID
+event :delete_relationship_answers,
+      :prepare_to_validate, on: :update, trigger: :required do
+  if !Card::Auth.always_ok?
+    # TODO: come up with better permissions scheme for this!
+    # maybe something like `perms: :admin` in the event def?
+    errors.add :answers, "only admins can bulk delete answers"
+  else
+    company = company_from_params
+    if !company
+      errors.add :company, "params must specify valid company"
+    else
+      delete_answers_for_company company
+      true
+    end
+  end
```

Comment:
too much error handling noise in my opinion. something like `with_company_from_params do ...`?